Source/Modules/app_view_models/Edit/Plugins/Sampler/SynthSamplerState.cpp

```diff
@@ -0,0 +1,52 @@
+#include "SynthSamplerState.h"
+namespace app_view_models {
+
+SynthSamplerState::SynthSamplerState(juce::ValueTree parent, int size)
+    : synthSamplerValue(
+          parent.getOrCreateChildWithName(IDs::synthSamplerStateID, nullptr)),
+      listSize(size) {
+    jassert(synthSamplerValue.hasType(IDs::synthSamplerStateID));
+    synthSamplerValue.addListener(this);
+
+    std::function<juce::String(juce::String)> noneConstrainer =
+        [this](juce::String param) { return param; };
+
+    curFilePath.setConstrainer(noneConstrainer);
```

Comment:
this should probably be removed, the constrainer should only be needed if there are indeed necessary constraints
```suggestion
```